docs/schema.md

```diff
@@ -289,6 +289,117 @@ you must also specify a `type`, described in
 Use the following types to indicate how the properties in your schema are
 treated by GCP Marketplace.
 
+### `schemaVersion`
+
+The version of the schema. If you want to
+[support managed updates for your app](https://cloud.google.com/marketplace/docs/partners/kubernetes-solutions/support-managed-updates), the value must be `v2`.
+
+### (Required) `applicationApiVersion`
+
+The version of the Application Custom Resource object. As of November 2019,
+the version is `v1beta1`.
+
+### (Required) `publishedVersion`
+
+The release version for the application, as a string. This must match the
+release tag on all your application images. For example, `'1.0.5'`.
+
+### `publishedVersionMetadata`
+
+Information about the version, shown to users in the GCP Console when they
+view their Kubernetes workloads.
+
+#### `releaseNote`
+
+Information about the release, using the
+[YAML folding style, with a block chomping indicator](https://yaml-multiline.info).
+For example:
+
+```yaml
+releaseNote: >-
+  Bug fixes and performance enhancements.
+```
+
+#### (Optional) `releaseTypes`
+
+A list of release types, which can be one or more of the following:
+
+- `Feature`
+- `BugFix`
+- `Security`. We recommend using `Security` only if the update addresses a
+  critical security issue. In the GCP Console, security updates are displayed
+  more prominently than other types of updates.
+
+#### `recommended`
+
+A boolean that indicates whether the update is recommended, such as for a
+security update. If `true`, users are encouraged to update as soon as possible.
+
+### `managedUpdates`
+
+Use this section to indicate that your deployer supports managed updates. If
+you omit this section, users must manually update their installations.
+
+#### `kalmSupported`
+
+If you want to support managed updates for your application, set this to
+`true`.
+
+### `images`
+
+New for schema v2. Declare the images for your application in this section.
+
+In the following, the following images are declared, in addition to the
```

Comment:
The explanation here (and the two image parameter paragraphs) pertain to the example YAML.

Essentially what's happening here is that:
- (1) The images are declared using the top level names under `images`. For example, the following defines 2 images: the main image and the `proxy` image. Both images share the same prefix (`gcr.io/your-project/your-company/your-app`), which is not specified in the schema but externally. This is important to note because the prefix can be anything, as long as the image structure is preserved.
```yaml
images:
  '': {}  # Note: empty curly braces is just YAML syntax required for the empty value.
  proxy: {}
```
- (2) They can be projected into the application templates as properties via the use of `properties` under each image name. An image can be projected into one or more properties. Each property can receive the full image name or some part of it. This is where the `type` comes in:
    - `FULL`: the entire image name, `gcr.io/your-project/your-company/your-app:1.0.1`.
    - `REGISTRY`: only the `gcr.io` part.
    - `REPO_WITHOUT_REGISTRY`: only the repo without the registry part or tag, i.e. `your-project/your-company/your-app`.
    - `REPO_WITH_REGISTRY`: the repo with the registry and without the tag, i.e. `gcr.io/your-project/your-company/your-app`
    - 'TAG`: the image tag, i.e. `1.0.1`.
Note that the `FULL` image name can always be constructed from `[REGISTRY][REPO_WITHOUT_REGISTRY]:[TAG]`, or `[REPO_WITH_REGISTRY]:[TAG]`.

Please help rewrite this content in a more reader friendly way :)